modules/serverless-ossm-disabling-network-policies.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * /serverless/integrations/serverless-ossm-setup.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-ossm-disabling-network-policies.adoc_{context}"]
+= Disabling the default network policies 
+
+The {ServerlessOperatorName} generates the network policies by default. To disable the default network policy generation, you can add the `serverless.openshift.io/disable-istio-net-policies-generation` annotation in the `KnativeEventing` and `KnativeServing` custom resources (CRs).
```

Comment:
I think the note about `manageNetworkPolicy: false` is applicable here as well and we should make it clear why we would disable default network policy generatio otherwise this section is pulled out of context. Something like this:

The {ServerlessOperatorName} generates the required network policies by default. When you configure ServiceMeshControlPlane with manageNetworkPolicy: false,  you must disable the default network policy generation to ensure proper event delivery. To disable the default network policy generation, you can add the `serverless.openshift.io/disable-istio-net-policies-generation` annotation in the `KnativeEventing` and `KnativeServing` custom resources (CRs).

Reply:
Added